plugins/ui/src/js/src/widget/WidgetHandler.test.tsx

```diff
@@ -275,3 +275,83 @@ it('updates the initial data only when widget has changed', async () => {
   unmount();
   expect(cleanup).toHaveBeenCalledTimes(1);
 });
+
+it('handles rendering widget error if widget is null (query disconnected)', async () => {
+  const widget1 = makeWidgetDescriptor();
+  const cleanup = jest.fn();
+  const addEventListener = jest.fn(() => cleanup);
```

Comment:
```suggestion
  const mockAddEventListener = jest.fn(
    (() => cleanup) as dh.Widget['addEventListener']
  );
```

Asserting the `addEventListener` type here would allow you to remove the `as any` assertion + eslint disable on lines 314-315

Reply:
Good suggestion, cleaned this up and the other usages in this file.